english/justice-league/justice-league.md

```diff
@@ -9,7 +9,8 @@ You send out a distress signal.
 Who will answer the call?
 
   - Superman?
-  - Batman?
+  - Batman
```

Comment:
You need to add the "?" back to "Batman" to make it so you haven't deleted a line and then ask @casey-collab to review it again.